Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -1408,6 +1408,7 @@
     <string name="errors_not_enough_device_storage">"You do not have sufficient free memory."</string>
     <!-- XTXT: error dialog - detailed text if there is error with Google API  -->
     <string name="errors_communication_with_api">"Error during communication with Google interface"</string>
+    <string name="errors_communication_with_api_test">"Test ABC"</string>
```

Comment:
Relict of some testing? Please remove it.

Reply:
Removed